### PR TITLE
add buildx-fallback to build-test-manylinux-aarch64

### PR DESCRIPTION
**Pull request by tpetr**
stopgap to make fork PRs succeed until we come up with a better plan

test plan:
- checks pass

/cc https://github.com/returntocorp/semgrep/pull/8281

### CODE REVIEW

**Review by cgdolan — APPROVED** [review]

File: .github/workflows/build-test-manylinux-aarch64.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@ jobs:
       id-token: write
     steps:
       - uses: docker/setup-qemu-action@v2
+      - uses: docker/setup-buildx-action@v2
```
[the remark this answers is not in the document]
Comment (tpetr) [reply .github/workflows/build-test-manylinux-aarch64.yaml:16]: Thanks Semgrep

File: .github/workflows/build-test-manylinux-aarch64.yaml [hunk 2]
```diff
@@ -22,6 +23,7 @@ jobs:
           platforms: linux/arm64
           outputs: type=docker,dest=/tmp/image.tar
           target: semgrep-wheel
+          buildx-fallback: true
```
Comment (aryx) [inline .github/workflows/build-test-manylinux-aarch64.yaml:26]: please add a comment, saying we need this for forked PRs.
Otherwise there's no way we gonna remember it in 6 months.

Comment (cgdolan) [reply .github/workflows/build-test-manylinux-aarch64.yaml:26]: The context about why this change was introduced is still captured in the commit message for future archaeologists to discover with `git blame`.

Comment (aryx) [reply .github/workflows/build-test-manylinux-aarch64.yaml:26]: No! We don't want to force people to use git blame to understand why the code is written the way it is. The code should be readable and understandable without having to do some archeology!

Comment (aryx) [reply .github/workflows/build-test-manylinux-aarch64.yaml:26]: We try hard to follow this rule in the semgrep code, so we should do the same for the GHA workflows.

Comment (tpetr) [reply .github/workflows/build-test-manylinux-aarch64.yaml:26]: I didn't articulate this well in the PR body, but the core issue here is that I intended to (but forgot) to set `buildx-fallback: true` when I created this workflow. This wasn't a new learning to fix a specific problem (depot could fail for any number of reasons), so I'm more inclined to direct folks to the docs instead of adding a one-off comment: https://github.com/depot/build-push-action#inputs

Comment (aryx) [reply .github/workflows/build-test-manylinux-aarch64.yaml:26]: FYI I've added some notes about comments vs git blame in https://www.notion.so/semgrep/How-we-code-together-ea3b47c3f3f04e57bd9f6167caff1335#1a28f23f4dd74c2488b1517bbc640e11
